manifests/runtime.yaml

```diff
@@ -25,3 +25,7 @@ spec:
     - name: sealed-secrets
       type: kustomize
       url: github.com/codefresh-io/cli-v2/manifests/sealed-secrets
+    - name: frpc
+      type: helm
+      url: github.com/codefresh-io/cli-v2/manifests/frpc?ref=CR-14253-ingressless-flag
```

Comment:
you are going to change that later to remove ref?

Reply:
yeah, it will be completely different. i am talking to Ilia and hopefully we will change it tonight, or they will change it tomorrow morning, to the "real" repository.